dds/DCPS/transport/rtps_udp/RtpsSampleHeader.cpp

```diff
@@ -264,7 +264,7 @@ RtpsSampleHeader::into_received_data_sample(ReceivedDataSample& rds)
           if (rtps.inlineQos[i]._d() == PID_KEY_HASH) {
             rds.sample_.reset(new ACE_Message_Block(20));
             // TODO(iguessthislldo) Convert to Encoding
```

Comment:
```suggestion
```

I'm fine with not doing this one since it's a weird case.

Reply:
It doesn't fit the pattern and the comment mention Encoding rather than encapsulation. I will cut the TODO line out.

Reply:
Yea I think I meant to say "encapsulation" because that's definitely what the line is setting.